src/annotator/integrations/pdf-metadata.js

```diff
@@ -133,28 +133,33 @@ export class PDFMetadata {
    */
   getMetadata() {
     return this._loaded.then(app => {
-      let title = document.title;
+      const url = getPDFURL(app);
 
+      let title;
       if (
         app.metadata &&
         app.metadata.has('dc:title') &&
         app.metadata.get('dc:title') !== 'Untitled'
       ) {
         title = /** @type {string} */ (app.metadata.get('dc:title'));
-      } else if (app.documentInfo && app.documentInfo.Title) {
+      } else if (app.documentInfo?.Title) {
         title = app.documentInfo.Title;
+      } else if (app._contentDispositionFilename) {
+        title = app._contentDispositionFilename;
```

Comment:
This is a private property that was recently renamed from `contentDispositionFilename`. I want to find a non-private way to get this.